src/handlers/push_message.rs

```diff
@@ -91,7 +171,18 @@ pub async fn handler(
             last_recieved_at = %notification.last_received_at,
             "notification has already been received"
         );
-        return Ok(Response::new_success(StatusCode::OK));
+
+        #[cfg(feature = "analytics")]
+        {
+            analytics.success = true;
```

Comment:
Is that a third type of item? In that case, maybe a third type in addition to 'request' and 'response; could cover this?

Reply:
Success is redundant as pointed out above, its been removed